runtime/src/bank.rs

```diff
@@ -9302,15 +9284,12 @@ pub(crate) mod tests {
         assert_ne!(bank1.capitalization(), bank0.capitalization());
 
         // verify the inflation is represented in validator_points *
-        let paid_rewards = bank1.capitalization()
-            - bank0.capitalization()
-            - bank1_sysvar_delta()
-            - new_epoch_sysvar_delta();
-
-        let rewards = bank1
-            .get_account(&sysvar::rewards::id())
-            .map(|account| from_account::<Rewards, _>(&account).unwrap())
-            .unwrap();
+        let paid_rewards = bank1.capitalization() - bank0.capitalization() - bank1_sysvar_delta();
+
+        let rewards = Rewards {
+            validator_point_value: 6625.150397619048,
+            unused: 0.0,
+        };
```

Comment:
Eventually this `Rewards` struct is going to be gone completely so I'd prefer if we don't use it here. In this case, I think we can ignore validator points altogether and directly compute the inflation rewards (probably need to factor out that logic from `update_rewards_with_thread_pool`) to use in the test.

Reply:
Cool, thanks for chiming in, Justin. I took a crack at calculating expected inflation in the unit test and updated this PR. Let me know what you think